src/examples/rest/src/admin-ui/custom-fields/index.ts

```diff
@@ -8,7 +10,23 @@ customFields.set('Task', [
 		type: 'custom',
 		index: 3,
 		component: Link,
-
 		hideOnDetailForm: true,
 	},
 ]);
+
+customFields.set('Credential', [
+	{
+		name: 'password',
+		type: 'custom',
+		component: PasswordComponent,
```

Comment:
To have a password field you have a custom field with a with a password component? Why not a password field as its own component?

Reply:
We could always convert this to a password field if we thought that was value in that.

Reply:
All the ui components to do with auth are in a separate package for now.